CMakeLists.txt

```diff
@@ -174,15 +180,9 @@ set(LINKER_FLAGS_RELEASE )
 set(EXTRA_LIBS )
 
 if(WIN32)
-    set(CPP_DEFS ${CPP_DEFS} _WIN32 NOMINMAX WIN32_LEAN_AND_MEAN)
-    if(MINGW)
-        set(CPP_DEFS ${CPP_DEFS} __USE_MINGW_ANSI_STDIO)
```

Comment:
This needs to be set to make printf and other stdio functions use MinGW's standards-compliant versions instead of the ones from Windows' ancient msvcrt.dll. It is intended for users to set it.

Reply:
This was the case in the past. Now it is automatically set in mingw32 and mingw64 which use `msvcrt.dll`. It should not be set for all other [environments](https://www.msys2.org/docs/environments/) that use `ucrt`. I even asked this question in their discord and they confirmed it.